docs/server/source/quickstart.md

```diff
@@ -57,6 +57,13 @@ I. Run BigchainDB Server:
 $ bigchaindb start
 ```
 
+J. Verify BigchainDB Server setup by visiting the BigchainDB Root Url in your browser (if not changed during installation):
```

Comment:
"URL" is an acronym and is still written as such, i.e. with capital letters for all three letters.

Why would the root URL be changed during installation? The whole point of a Quickstart is that they are on rails, not doing anything fancy or weird. Please remove the trailing phrase inside parentheses.
```text
J. Verify BigchainDB Server setup by visiting the BigchainDB Root URL in your browser:
```